.changeset/smart-buses-knock.md

```diff
@@ -0,0 +1,5 @@
+---
+"@justeattakeaway/pie-web-components-core": patch
```

Comment:
Should this be `minor`?

Reply:
https://www.youtube.com/watch?v=hsornYQ7Uvs